force-app/main/default/aura/auraEmbeddedLWC/auraEmbeddedLWC.css

```diff
@@ -1,36 +1,36 @@
 .THIS img {
-    width: 80px;
-    height: 80px;
-    border-radius: 50%;
-    margin-bottom: 8px;
+    width: token(squareIconLargeBoundaryAlt);
+    height: token(squareIconLargeBoundaryAlt);
+    border-radius: token(borderRadiusCircle);
+    margin-bottom: token(varSpacingXSmall);
 }
 
 .THIS .slds-card__footer {
-    text-align: left;
-    color: #706e6b;
+    color: rgb(112, 110, 107);
```

Comment:
@satyasekharcvb it was pointed out to me by @pozil that we shouldn't override slds styles. 

There's actually an existing utility class for this color that can be applied right to markup: `slds-color__text_gray-9`. See [here](https://github.com/trailheadapps/lwc-recipes/blob/db1d23f9064538eae2395c8ca9fa75aeada13256/force-app/main/default/aura/lmsPublisherAuraComponent/lmsPublisherAuraComponent.cmp#L38).

In this case, these aura components are marked to be removed from the project anyway in #287, so don't worry about fixing. I just thought you'd like to know.

Reply:
For all the LWC, I have used only the design tokens which are GA and the SLDS utility classes that are available in the latest SLDS release notes.